marketplace-kit-archive.js

```diff
@@ -17,11 +17,15 @@ const checkDirectory = directoryPath => {
 };
 
 const makeArchive = (path, directory) => {
-  checkDirectory(directory);
+  //checkDirectory(directory);
+
+  logger.Info(path);
```

Comment:
If it is final version of this line i would like to see at least a title of what this path is showing, but TBH i dont see much of a value in it for end user